index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
```

Comment:
Hmm, I think this file (and the ones below it) are build artifacts and aren't supposed to be checked in...